client/blocks/login/social.jsx

```diff
@@ -40,11 +40,17 @@ class SocialLoginForm extends Component {
 	handleGoogleResponse = ( response ) => {
 		const { onSuccess, redirectTo } = this.props;
 
-		if ( ! response.Zi || ! response.Zi.id_token ) {
+		if ( ! response.Zi || ! response.Zi.access_token || ! response.Zi.id_token ) {
 			return;
 		}
 
-		this.props.loginSocialUser( 'google', response.Zi.id_token, redirectTo )
+		const socialInfo = {
+			service: 'google',
+			access_token: response.Zi.access_token,
```

Comment:
Well we could but this format has the advantage of being able to be sent directly to our social login endpoint without any formatting, and it isn't used in any of our components. Let's be lazy for now and try to make it work first ;)